Src/FluentAssertions/Execution/AssertionScope.cs

```diff
@@ -14,60 +10,40 @@ namespace FluentAssertions.Execution;
 /// Represents an implicit or explicit scope within which multiple assertions can be collected.
 /// </summary>
 /// <remarks>
-/// This class is supposed to have a very short lifetime and is not safe to be used in assertions that cross thread-boundaries
+/// This class is supposed to have a very short life time and is not safe to be used in assertion that cross thread-boundaries
```

Comment:
I think this should be reverted, because "lifetime" != "life time" and I think former is correct here...
same with "assertion"